### PR TITLE
Feat/add elsa endpoint

### PR DESCRIPTION
**Pull request by montenegrodr**
This PR also includes the removal of non-anymore supported endpoints such as

- Batch
- Related
- Microformats
- Unsupervised

### CODE REVIEW

**Comment by montenegrodr** [comment]
[(1) [TextApi / ELSA] Node.JS SDK](https://trello.com/c/KPN34T2D/601-1-textapi-elsa-nodejs-sdk)

File: textapi.js [hunk 1]
```diff
@@ -72,6 +72,25 @@ function AYLIENTextAPI(options) {
...
+  /**
+   * textapi.entityLevelSentiment
+   *
+   * @desc Returns the sentiment towards entities on given text.
+   *
+   * @memberof! aylien_textapi(v1)
+   * @instance
+   *
+   * @param     {object}    params - Parameters for request
+   * @param     {string}    params.text - Text to analyze
```
Comment (afshinm) [inline textapi.js]: we don't have `text` and `url` in the function arguments.

Comment (montenegrodr) [reply textapi.js]: Yeah, `params` is the argument.. which have `text` and `url` properties. Is this completely wrong cause we have the same pattern for all other API functions.

Comment (afshinm) [reply textapi.js]: yes, that's wrong but if we have the same pattern for others, let's continue for now and maybe adding a card to address this issue later.

Comment (montenegrodr) [reply textapi.js]: Will do, may I have your :+1: if applicable? :)

**Pull request merged**